executor/window.go

```diff
@@ -481,7 +489,12 @@ func (p *rangeFrameWindowProcessor) appendResult2Chunk(ctx sessionctx.Context, r
 			if slidingWindowAggFunc != nil && initializedSlidingWindow {
 				err = slidingWindowAggFunc.Slide(ctx, rows, lastStart, lastEnd, shiftStart, shiftEnd, p.partialResults[i])
 			} else {
-				_, err = windowFunc.UpdatePartialResult(ctx, rows[start:end], p.partialResults[i])
+				if minMaxSlidingWindowAggFunc, ok := windowFunc.(aggfuncs.MaxMinSlidingWindowAggFunc); ok {
+					minMaxSlidingWindowAggFunc.SetWindowStart(start)
+					_, err = minMaxSlidingWindowAggFunc.UpdatePartialResult(ctx, rows[start:end], p.partialResults[i])
+				} else {
+					_, err = windowFunc.UpdatePartialResult(ctx, rows[start:end], p.partialResults[i])
```

Comment:
Same logic as in `rowFrameWindowProcessor`

Reply:
```
if slidingWindowAggFunc != nil && initializedSlidingWindow {
	err = slidingWindowAggFunc.Slide(ctx, rows, lastStart, lastEnd, shiftStart, shiftEnd, p.partialResults[i])
} else {
	if minMaxSlidingWindowAggFunc, ok := windowFunc.(aggfuncs.MaxMinSlidingWindowAggFunc); ok {
		minMaxSlidingWindowAggFunc.SetWindowStart(start)
	}
	_, err = windowFunc.UpdatePartialResult(ctx, rows[start:end], p.partialResults[i])
}
```

It looks more clean, if we remove the `UpdatePartialResult` and `ResetPartialResult` of the `MaxMinSlidingWindowAggFunc` interface

Reply:
Great advice! Now this PR looks much more concise.